scripts/refresh-code-excerpts.sh

```diff
@@ -14,6 +14,9 @@ if [[ $1 == '-h' || $1 == '--help' ]]; then usage; fi
 
 [[ -z "$NGIO_ENV_DEFS" ]] && . $rootDir/scripts/env-set.sh
 
+# Until site-www has its own schredder, use the one from site-webdev
```

Comment:
"schredder" must be the German form of "shredder"!

Reply:
Oups! :)

Reply:
Fixed (along with another addition I needed to make).